scripts/maintenance/utils.py

```
@@ -0,0 +1,21 @@
+    """Executes a command securely using list-based arguments and shell=False."""
+    try:
+        result = subprocess.run(
+            args,
+            shell=False,
+            check=False,
+            capture_output=True,
+            text=True
+        )
+        if result.returncode == 0:
+            return result.stdout.strip()
+        return None
```

Ruff (0.15.13)

[warning] Consider moving this statement to an `else` block

(TRY300)